src/query/sql/src/planner/semantic/type_check.rs

```diff
@@ -741,6 +742,10 @@ impl<'a> TypeChecker<'a> {
                 {
                     if let Some(udf) = self.resolve_udf(*span, func_name, args).await? {
                         return Ok(udf);
+                    } else if let Some(async_function) =
+                        self.resolve_async_function(*span, func_name, args).await?
```

Comment:
Where filter can contains a sync function? @b41sh

Reply:
seems `async_function` can only used in insert values.